packages/ui-components/src/components/switch.ts

```diff
@@ -25,7 +25,9 @@ export class Switch extends Widget {
     track.className = 'jp-switch-track';
     track.setAttribute('aria-hidden', 'true');
 
-    this._button.appendChild(this._label);
+    if (this._label.textContent !== '') {
```

Comment:
Could it be that this change has a side effect on the mode switch in the status bar, resulting in not showing the "Mode" label?

![image](https://user-images.githubusercontent.com/591645/101616751-115b0e80-3a10-11eb-9b7d-5d56f1b17fa9.png)

Reply:
Oh good point I didn't notice that effect - I added that because it was adding an empty label to the debugger switch and creating some weird css: 
![image](https://user-images.githubusercontent.com/6673460/101656128-5c752180-3a10-11eb-989d-210f9cc64051.png)
But I just undid this and changed the css to fix it instead. 
![image](https://user-images.githubusercontent.com/6673460/101657295-b5918500-3a11-11eb-8d19-0701b3dc8fc5.png)